>From a2cf1cc7f68cffc7cb1ed91583daf5a2167a00d1 Mon Sep 17 00:00:00 2001
From: Thijs Kinkhorst <thijs@debian.org>
Date: Thu, 6 Jun 2013 16:40:17 +0200
Subject: [PATCH] Replace footer 'Run by Debian' with 'Run on Debian'.

To make it more clear that it's not run by Debian the project, but on
Debian the OS.
---
 debian/patches/debianize.diff |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/patches/debianize.diff b/debian/patches/debianize.diff
index 318cb83..7d8bc24 100644
--- a/debian/patches/debianize.diff
+++ b/debian/patches/debianize.diff
@@ -47,7 +47,7 @@ Index: dokuwiki/lib/tpl/default/footer.html
  
    <a <?php echo $tgt?> href="http://jigsaw.w3.org/css-validator/check/referer?profile=css3" title="Valid CSS"><img src="<?php echo DOKU_TPL; ?>images/button-css.png" width="80" height="15" alt="Valid CSS" /></a>
  
-+  <a <?php echo $tgt?> href="http://www.debian.org" title="Run by Debian"><img src="<?php echo DOKU_TPL; ?>images/button-debian.png" width="80" height="15" alt="Run by Debian" /></a>
++  <a <?php echo $tgt?> href="http://www.debian.org" title="Run on Debian"><img src="<?php echo DOKU_TPL; ?>images/button-debian.png" width="80" height="15" alt="Run on Debian" /></a>
 +
    <a <?php echo $tgt?> href="http://dokuwiki.org/" title="Driven by DokuWiki"><img src="<?php echo DOKU_TPL; ?>images/button-dw.png" width="80" height="15" alt="Driven by DokuWiki" /></a>
  
-- 
1.7.10.4

